ocaml/idl/datamodel.ml

```diff
@@ -608,6 +624,8 @@ let _ =
     ~doc:"The operation you requested cannot be performed because the specified PIF has FCoE SR in use." ();
   error Api_errors.pif_unmanaged [ "PIF" ]
     ~doc:"The operation you requested cannot be performed because the specified PIF is not managed by xapi." ();
+  error Api_errors.pif_is_not_sriov_capable [ "PIF" ]
+    ~doc:"You try to enable network SR-IOV on an incapable PIF" ();
```

Comment:
`try -> tried` but I would write something like `"The selected PIF is not capable of network SR-IOV."`